### PR TITLE
Finding the Numbers

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Program to find the Unique numbers in an array out of which two numbers occur exactly once and are distinct.
You need to find the other two numbers and print them in ascending order.

Also Taken help from geeksforgeeks

### Checklist

- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#922

### CODE REVIEW

**Comment by narayan2111** [comment]
kindly review this PR @iamrajiv @plazzy99 @Kajol-Kumari

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
https://www.geeksforgeeks.org/find-unique-pair-array-pairs-numbers/

code is exactly same with comments and all

**Comment by narayan2111** [comment]
> https://www.geeksforgeeks.org/find-unique-pair-array-pairs-numbers/
> 
> code is exactly same with comments and all

Updated the whole code and comments but algorithm is same.
Also I don't find any algorithm better than this.

**Comment by narayan2111** [comment]
> https://www.geeksforgeeks.org/find-unique-pair-array-pairs-numbers/
> 
> code is exactly same with comments and all

@plazzy99 
Updated the whole code and comments but algorithm is same.
Also I added geeksforgeeks in comment from where i had taken help

**Comment by narayan2111** [comment]
Kindly review this PR @Kajol-Kumari @iamrajiv @plazzy99

File: C/other/Find_the_numbers.c [hunk 1]
```diff
@@ -0,0 +1,61 @@
+//Program to find the Unique numbers in an array out of which two numbers occur exactly once and are distinct.
```
Comment (Kajol-Kumari) [inline C/other/Find_the_numbers.c:1]: please use multiline comment here.

**Review by Kajol-Kumari — APPROVED** [review]
@narayan2111 Minor comment, Rest LGTM :+1:

**Review by iamrajiv — APPROVED** [review]

**Comment by narayan2111** [comment]
Kindly review this PR @plazzy99

**Review by vatsalkesarwani12 — APPROVED** [review]

**Pull request merged**